pulsar-broker/src/main/java/com/yahoo/pulsar/broker/admin/PersistentTopics.java

```diff
@@ -155,6 +156,46 @@
     }
 
     @GET
+    @Path("/{property}/{cluster}/{namespace}/partitioned")
+    @ApiOperation(value = "Get the list of partitioned topics under a namespace.", response = String.class, responseContainer = "List")
+    @ApiResponses(value = { @ApiResponse(code = 403, message = "Don't have admin permission"),
+            @ApiResponse(code = 404, message = "Namespace doesn't exist") })
+    public List<String> getPartitionedTopicList(@PathParam("property") String property, @PathParam("cluster") String cluster,
```

Comment:
@rdhabalia 
I implemented this API under `persistent` similar like I referenced [getList](https://github.com/yahoo/pulsar/blob/master/pulsar-broker/src/main/java/com/yahoo/pulsar/broker/admin/PersistentTopics.java#L124).
getDestinations and getList  are completely same excepting for sorted or not.

@merlimat @saandrews 
Which is the better whether this API belong under `namespaces` or `persistent` ?